frontend/.storybook/theme.js

```diff
@@ -0,0 +1,30 @@
+import { create } from '@storybook/theming/create';
```

Comment:
I assume this theme is to theme storybook its self, would be cool if we could use the clutch theme one day.

Reply:
Yeah that would be ideal. I think the best we can get is reusing the same palette sine the keys don't map.